### PR TITLE
chore(tests): #13 - Prepare dockerfile running unit and integration tests

### PR DESCRIPTION
**Pull request by grzesuav**
Initial idea, not sure about dockerfile name

### CODE REVIEW

**Comment by grzesuav** [comment]
to verify, run `docker build -t metacontroller:test -f Tests.dockerfile .`

**Comment by AmitKumarDas** [comment]
Is this file already a part of GitHub actions

File: Tests.dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM golang:1.10 AS build
```
Comment (AmitKumarDas) [inline Tests.dockerfile:1]: Google search indicates `Dockerfile.test` may be a better name instead of `Tests.dockerfile`

Comment (AmitKumarDas) [reply Tests.dockerfile:1]: Is this used in GitHub actions?

Comment (grzesuav) [reply Tests.dockerfile:1]: not yet, it is initial proposal. will change that and update actions as well

**Comment by grzesuav** [comment]
@AmitKumarDas implemented it, not sure about logging level probably it should be lowered to 1 ?

**Pull request merged**